### DISCUSSION TITLE
Multiple Secret fields

### DISCUSSION THREAD

**Discussion by MaximBar** [keepassxreboot/keepassxc#8269] [Ideas]
I would like to suggest a feature request, to have the option to add multiple "password\secret" fields with customized names to entries.
For the example if I add a Gmail account, the following fields should be "secret":
Password
App Password
Otp Secret
So it would be great if we could manually have the option to add new fields and specify if we want them to be secret or regular fields.

**Comment by droidmonkey** [comment]
Go to the advanced tab and add additional attributes. You can make certain ones as protected. For OTP, setup TOTP using the gui. Check out the user guide for more information.

**Reply by MaximBar** [reply]
Thank you, it does help.
But it would be nice to see all in one main entry page without going to the advanced tab.
